server/local.ts

```diff
@@ -22,13 +23,15 @@ export function handler(env: Env) {
           return await provider.handle(request, pathInfo)
         } catch (e) {
           if (e instanceof Response) return e
-          return f.serverError(Deno.inspect(e))
+          const eText = Deno.inspect(e)
+          if (env.dbg) console.error(eText)
+          return f.serverError(eText)
```

Comment:
This is shorter and will give colors
```suggestion
          if (env.dbg) console.error(e)
          return f.serverError(Deno.inspect(e))
```